src/test/java/com/celements/model/classes/fields/StringFieldTest.java

```diff
@@ -20,8 +20,7 @@
   @Before
   public void setUp() throws Exception {
     super.setUp();
```

Comment:
please do not override the default setUp method. Instead add a different named method with Before annotation.